rpc/src/rpc_subscriptions.rs

```diff
@@ -2700,7 +2700,9 @@ pub(crate) mod tests {
         bank_forks.write().unwrap().insert(bank1);
         let bank2 = Bank::new_from_parent(&bank0, &Pubkey::default(), 2);
         bank_forks.write().unwrap().insert(bank2);
-        let alice = Keypair::new();
+
+        // we need a pubkey that will pass its rent collection slot so rent_epoch gets updated to max since this account is exempt
+        let alice = Keypair::from_base58_string("sfLnS4rZ5a8gXke3aGxCgM6usFAVPxLUaBSRdssGY9uS5eoiEWQ41CqDcpXbcekpKsie8Lyy3LNFdhEvjUE1wd9");
```

Comment:
What is the rent-collection slot for this account? I'm assuming 1 or 2...?

Reply:
I don't know. I put this in a loop generating random keys until one had a max value for rent_epoch. I don't know how many slots there are in the epoch either. It is a test setup.

Reply:
Are we sure it will be collected within the test slots every time, then?

Reply:
The # of slots in the epoch and the partition index of this particular pubkey within that # of slots/epoch is deterministic.

Reply:
the randomness was in the key generation. Sometimes it fell in the required partition index(es) and usually it didn't.

Reply:
Okay, great, thanks for the assurance :)
:shipit: